local/local-flowctl-config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I assume these are specially crafted to live for a very long time, or something like that?

Reply:
Yeah, the `public_token` and `endpoint` don't really change, so this should be a one-time operation to copy this file to your config directory, and from that point on we'll merge changes to it as you (re-)authenticate.